demo/src/pages/demos/character.tsx

```diff
@@ -0,0 +1,284 @@
+/* eslint-disable max-classes-per-file */
+import React, { Component } from "react";
+
+import Axes, {
+  PanInput,
+  MoveKeyInput,
+  PinchInput,
+  WheelInput,
+} from "../../../../packages/react/src";
```

Comment:
It is recommended not to use react-axes.

Reply:
Let's add a new demo in the next patch update